src/heads-up/lib/find-meta.js

```diff
@@ -42,6 +42,35 @@ export function findMetaProperty(head, name) {
   return item ? item.content : null
 }
 
+export function findXMLElement(text, tagName) {
+  const parser = new DOMParser()
+  const xml = parser.parseFromString(text, 'text/xml')
+
+  const tags = Object.values(xml.documentElement.childNodes)
```

Comment:
I am quite sure you can replace both `filters` with a single `find()`.

Reply:
I need an Array of `childNodes` that were found, and `find()` just returns the element.